src/server/main.ts

```diff
@@ -195,11 +195,60 @@ async function handlefeatureToggles({ featureToggles, extensionSettings }: Aurel
     console.log(`${sourceDirectory  }/${  settings.extensionSettings.pathToAureliaProject[0]}`);
 
     try {
-      const fileProcessor = new ProcessFiles();
-      await fileProcessor.processPath(extensionSettings);
-      aureliaApplication.components = fileProcessor.components;
-      console.log('>>> 2. The extension found this many components:');
-      console.log(aureliaApplication.components.length);
+      const updateAureliaComponents = async () => {
+        const fileProcessor = new ProcessFiles();
+        await fileProcessor.processPath(extensionSettings);
+        aureliaApplication.components = fileProcessor.components;
+
+        console.log('>>> The extension found this many components:');
+        console.log(aureliaApplication.components.length);
+      };
+
+      const configPath = ts.findConfigFile(
+        /* searchPath */ "./",
+        ts.sys.fileExists,
+        "tsconfig.json"
+      );
```

Comment:
```suggestion
      const configPath = ts.findConfigFile(
        /* searchPath */ "./",
        ts.sys.fileExists,
        "tsconfig.json"
      );
      
      if (configPath === undefined) {
        configPath = '../../tsconfig.json' // use config file from the extension as default
      }
```

Reply:
Good idea. Perhaps we should have a dedicated config in the repo which is easier to track and change.